cookbook/import_export/product-import-from-xml-file.rst

```diff
@@ -41,21 +41,38 @@ Register the bundle in AppKernel:
 Configure the Job
 -----------------
 
-Configure a job in ``Resources/config/batch_jobs.yml``:
+Configure a job in ``Resources/config/jobs.yml``:
 
-.. literalinclude:: ../../src/Acme/Bundle/XmlConnectorBundle/Resources/config/batch_jobs.yml
+.. literalinclude:: ../../src/Acme/Bundle/XmlConnectorBundle/Resources/config/jobs.yml
    :language: yaml
    :linenos:
-   :lines: 1-13
-
-Here we create an import job which contains a single step: `import`.
+   :lines: 2-11
 
 The default step is ``Akeneo\Component\Batch\Step\ItemStep``.
 
 An item step is configured with 3 elements, a reader, a processor and a writer.
 
+Here is the definition of the Step:
+
+.. literalinclude:: ../../src/Acme/Bundle/XmlConnectorBundle/Resources/config/steps.yml
+   :language: yaml
+   :linenos:
+   :lines: 2-11
+
 Here, we'll use a custom reader ``acme_xml_connector.reader.file.xml_product`` but we'll continue to use default processor and writer.
```

Comment:
"to use the default" (I know you didn't change this, but could you please fix it?)